target/classes/META-INF/sun-jaxb.episode

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
please exclude the target folder from git :) This can be generated simply by running a build anytime